### DISCUSSION TITLE
Single Row Resource

### DISCUSSION THREAD

**Discussion by austyuzhaninov** [moonshine-software/moonshine#218] [Ideas]
День добрый!

Есть предложение реализовать простой и красивый параметр ресурса а-ля SingleRowResource

```
class SuperResource extends Resource
{
    protected static bool $singleRowResource = true;
}

```

Нечто подобное есть в Nova
https://novapackages.com/packages/daniel-de-wit/nova-single-record-resource

Такой параметр удобно использовать например в настройках.

**Comment by lee-to** [comment]
Сейчас это можно легко реализовать,в нашем чате телеграма я скидывал пример с ресурсом который просто выводит лог

**Reply by austyuzhaninov** [reply]
> Сейчас это можно легко реализовать,в нашем чате телеграма я скидывал пример с ресурсом который просто выводит лог

Подписался, спасибо)
